plugins/orchestrator-backend/src/service/DataIndexService.ts

```diff
@@ -91,25 +98,13 @@ export class DataIndexService {
   }
 
   public async getWorkflowDefinitions(): Promise<WorkflowInfo[]> {
-    const QUERY = `
-        query ProcessDefinitions {
-            ProcessDefinitions {
-                id
-                name
-                version
-                type
-                endpoint
-                serviceUrl
-            }
-        }
-      `;
-
+    const graphQlQuery = buildGraphQlQuery({
+      type: 'ProcessDefinitions',
+      queryBody: 'id, name, version, type, endpoint, serviceUrl',
+    });
+    this.logger.debug(`GraphQL query: ${graphQlQuery}`);
     this.logger.info(`getWorkflowDefinitions() called:  ${this.dataIndexUrl}`);
-    const result = await this.client.query(QUERY, {});
-
-    this.logger.debug(
-      `Get workflow definitions result: ${JSON.stringify(result)}`,
-    );
```

Comment:
You have removed this log probably during conflict resolution. Please put it back.

Reply:
[fb3b435](https://github.com/janus-idp/backstage-plugins/pull/1251/commits/fb3b435c53c7d1cfbb5ae51018affccdeede9c2d)